### ISSUE TITLE
A11Y of your website

### ISSUE THREAD

**Issue by jhermann** [mikolajdobrucki/ikonate#7]
Hi, you stress accessibility, but your main text on the site is #828398 on almost-white. I'd reconsider making the light grey way more black, like below #333.

**Comment by mikolajdobrucki** [comment]
Thanks for the feedback! Definitely a very valid point, I will include it in the next update

**Comment by mikolajdobrucki** [comment]
We deployed a fix to the page. The contrast should be much better now.
